src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/HomeController.kt

```diff
@@ -24,4 +33,18 @@ class HomeController(
         model["user"] = userProfile.id
         return "index"
     }
+
+    @GetMapping("/metrics", produces = ["text/plain"])
+    fun metrics(): ResponseEntity<String>
+    {
+        val workerStatus = hintrAPIClient.get("hintr/worker/status").body
+        val data = objectMapper.readTree(workerStatus)["data"]
+        val statuses = mutableMapOf("busy" to 0, "idle" to 0, "paused" to 0, "exited" to 0, "lost" to 0)
+        data.fields().forEach {
```

Comment:
For ease of use with prometheus, did you want to add a "live" workers field here, being the sum of busy, idle and paused?

Reply:
Sure